prisma/schema.prisma

```diff
@@ -140,9 +141,8 @@ model ComicIssue {
   description          String             @default("")
   flavorText           String             @default("")
   cover                String             @default("")
-  signedCover          String             @default("")
-  usedCover            String             @default("")
-  usedSignedCover      String             @default("")
+  featuredCover        FeaturedCover[]
+  stateFullCover       StateFullCover[]                
```

Comment:
same thing, these columns should be in plural.

Class name is always in singular, array is in plural.
`FeaturedCover` is a class and represents a single cover item. `FeaturedCover[]` is an array of featured covers and thus the variable should be named `featuredCovers`

Also, stateFull is actually **stateful** (lowercase and a single "L" character)